x-pack/plugins/task_manager/server/config.ts

```diff
@@ -109,6 +110,9 @@ export const configSchema = schema.object(
         defaultValue: {},
       }),
     }),
+    monitored_stats_warn_drift_in_seconds: schema.number({
```

Comment:
I added it there and also renamed it to avoid needing to define a new term for users! I'm not sure how to add things to the docker/cloud whitelist - is that a file in the Kibana repo?

Reply:
the docker whitelist is in this repo, and is listed in the PR template (at least when I create new PRs) - https://github.com/elastic/kibana/blob/master/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

the cloud whitelist is in a private repo; I'll DM you the info ...